bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BMap.java

```diff
@@ -282,6 +282,43 @@ public String stringValue() {
         }
     }
 
+    /**
+     * String value ignoring the access restrictions.
+     * Non-public fields of an object will also be stringified.
+     *
+     * @return string value
+     */
+    public String stringValueIgnoreAccessRestriction () {
```

Comment:
Shall we come up with a better method name here?

Reply:
Ack; Renamed the method.